api/openapi/v2/system-agent.yaml

```diff
@@ -0,0 +1,625 @@
+openapi: 3.0.0
+info:
+  title: Edgex Foundry - System Management Agent API
+  description: This is the definition of the API for the System Management Agent service in the EdgeX Foundry IOT microservice platform. System Management Agent is responsible for management of EdgeX microservices and supported infrastructure.
+  version: 2.x
+
+servers:
+  - url: "http://localhost:48090/api/v2"
+    description: URL for local development and testing
+
+components:
+  schemas:
+    BaseRequest:
+      description: "Defines basic properties which all use-case specific request DTO instances should support."
+      type: object
+      properties:
+        requestId:
+          description: "Uniquely identifies this request. For implementation, recommend this value be generated by the type's constructor."
+          type: string
+          format: uuid
+          example: "e6e8a2f4-eb14-4649-9e2b-175247911369"
+      required:
+        - requestId
+    BaseResponse:
+      description: "Defines basic properties which all use-case specific response DTO instances should support"
+      type: object
+      properties:
+        requestId:
+          description: "Uniquely identifies the request that resulted in this response."
+          type: string
+          format: uuid
+          example: "e6e8a2f4-eb14-4649-9e2b-175247911369"
+        statusCode:
+          description: "A numeric code signifying the operational status of the response."
+          type: integer
+        message:
+          description: "A field that can contain a free-form message, such as an error message."
+          type: string
+      required:
+        - requestId
+        - statusCode
+    ErrorResponse:
+      allOf:
+        - $ref: '#/components/schemas/BaseResponse'
+      description: "A response type for returning a generic error to the caller."
+    GetConfigRequest:
+      description: "A request associated with the /config endpoint."
+      allOf:
+        - $ref: '#/components/schemas/BaseRequest'
+      type: object
+      properties:
+        service:
+          description: "The service from which to obtain metrics"
+          type: string
+          example: "edgex-core-data"
+    GetConfigResponse:
+      allOf:
+        - $ref: '#/components/schemas/BaseResponse'
+      description: "Provides a response containing the configuration for the targeted service."
+      type: object
+      properties:
+        config:
+          description: "A string-ified representation of the service's configuration. For purposes of this specification, a string has been used since configuration structure differs from service to service."
+          type: string
+      required:
+        - config
+    HealthRequest:
+      allOf:
+        - $ref: '#/components/schemas/BaseRequest'
+      properties:
+        services:
+          description: "The services from which to obtain health information."
+          type: array
```

Comment:
Since it'd be possible in v2 to send multiple use-case requests in a single transport request, I wonder if we still need to support requesting for more than one service in a use-case request.